.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,61 @@
+## Descriptions
+
+## Check list before review
+
+🙏 Check(s) to perform _before_ asking for review for anyone.
+
+- [ ] Ensure the title complies with [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).
+  > Example: `feat: example [#00000000]`
+  >
+  > More details:
+  > https://wiki.worldr.co.uk/en/task-flow
+- [ ] Set the correct [flowfast link](#flowfast).
+- [ ] Fill in the description [for developers](#for-developers) and [for customers](#for-customers) if relevant.
+- [ ] Mark correct [types of changes](#types-of-changes).
+- [ ] Ensure there are tests, or an explanation as to why they are not needed.
+- [ ] Ensure there is documentation, or an explanation as to why it is not needed.
+- [ ] Rebase on top of `main` branch before review and before merging.
+- [ ] Ensure that CI/CD passed.
+- [ ] Review the changes by yourself.
+
+### For developers
+
+<!--- A description of what this pull request does. -->
```

Comment:
Perhaps this could help with consistency?

```suggestion
<!--- A description of what this pull request does as a short, bullet-point summary. -->
```

Reply:
@seanmc86 everything fixed